src/XMakeTasks/UnitTests/Microsoft.Build.Tasks.UnitTests.csproj

```diff
@@ -67,6 +67,16 @@
     <Compile Include="Touch_Tests.cs" />
     <Compile Include="TrustInfo_Tests.cs" />
     <Compile Include="WriteCodeFragment_Tests.cs" />
+    <Compile Include="AssignTargetPath_Tests.cs" />
+    <Compile Include="ConvertToAbsolutePath_Tests.cs" />
+    <Compile Include="CreateCSharpManifestResourceName_Tests.cs" />
+    <Compile Include="FindAppConfigFile_Tests.cs" />
+    <Compile Include="FindUnderPath_Tests.cs" />
+    <Compile Include="GenerateBindingRedirects_Tests.cs" />
+    <Compile Include="GetReferencePaths_Tests.cs" />
+    <Compile Include="ResolveCodeAnalysisRuleSet_Tests.cs" />
+    <Compile Include="ToolTaskExtension_Tests.cs" />
+
```

Comment:
Can you preserve alphabetical order in this list for hopefully-easier future merging?